CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.1.2 ##
+
+* Added withDiscoveryTimeout to GrpcTransportBuilder
+* Added query statistics to DataQueryResult
+* Fixed thread race in TopicService
```

Comment:
* Fixed Reader hang-up on server restart

The (harmless) race was in the first version of the fix itself :)
So it shouldn't be in change log

Reply:
Updated